### PR TITLE
feat(Duolingo): v3.2.0

### LINKED ISSUE #7787: Duolingo | www.duolingo.com - Support Alphabet learning pages
### Presence name

Duolingo

### Description

On the duolingo presence it does not display anything when you are doing exercises to learn new alphabets/letters for for example Russian. I would like support for these pages to be added if possible ❤
https://www.duolingo.com/characters

### Alternatives

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by veryCrunchy**
## Description 

+ added alphabet pages. resolves #7787
+ added practice pages
+ added all colors to progress bar
+ improved user page handling
+ improved lesson state handling

### CODE REVIEW

File: websites/D/Duolingo/metadata.json [hunk 1]
```diff
@@ -8,14 +8,6 @@
...
 			"name": "TheSlowly",
 			"id": "505071508851523599"
-		},
-		{
-			"name": "narpi",
-			"id": "321665259842830336"
-		},
-		{
-			"name": "Luke",
-			"id": "279622919242514432"
```
Comment (theusaf) [inline websites/D/Duolingo/metadata.json:18]: why remove these contributors?

File: websites/D/Duolingo/presence.ts [hunk 2]
```diff
@@ -73,91 +80,107 @@ const presence = new Presence({
...
-	return `${"🟩".repeat(completedSquares)}${"⬛".repeat(
-		size - completedSquares
-	)} ${progressPercentage}%`;
-}
-
-function handleLesson() {
-	user.inLesson = true;
+function handleLesson(_path: string | string[]) {
+	const progressBarElement =
+			document.querySelector('div[role="progressbar"]') ?? null,
```
Comment (theusaf) [inline websites/D/Duolingo/presence.ts:98]: `?? null` is somewhat redundant
```suggestion
			document.querySelector('div[role="progressbar"]'),
```

**Review by theusaf — CHANGES_REQUESTED** [review]

File: websites/D/Duolingo/presence.ts (hunk 2 above)
Comment (veryCrunchy) [reply websites/D/Duolingo/presence.ts:98]: No no, I remember it crying about a null check

Comment (veryCrunchy) [reply websites/D/Duolingo/presence.ts:98]: Will check when I'm home

Comment (veryCrunchy) [reply websites/D/Duolingo/presence.ts]: Had to put it in for something I scrapped, forgot to remove it

**Review by Bas950 — APPROVED** [review]

**Pull request merged**